src/main/java/org/simplify4u/plugins/PGPVerifyMojo.java

```diff
@@ -391,7 +411,7 @@ private Artifact resolveAscArtifact(Artifact artifact) throws MojoExecutionExcep
                 if (failNoSignature) {
                     getLog().error("No signature for " + artifact.getId());
                     throw new MojoExecutionException("No signature for " + artifact.getId());
-                } else {
+                } else if (!strictNoSignature) {
```

Comment:
call verifySignatureUnavailable in this place and throw exception - so only this place will be changed

Reply:
IIUC this will result in build failing at the first unlisted, unsigned artifact. So you lose the ability to list all unsigned artifacts, so the user can solve them in one go. Your thoughts?